components/hab/src/cli.rs

```diff
@@ -435,6 +430,7 @@ pub fn get() -> App<'static, 'static> {
             )
         )
         (subcommand: alias_apply)
+        (subcommand: alias_binlink)
```

Comment:
We try to keep the root level aliasing to a minimum. If you think we should add this one to the root we should probably cover it in another PR and have a discussion there

Reply:
Agreed, this can stay only under `hab pkg` for now, otherwise we start to massively widen the first-level subcommand space.